test/controllers/users_controller_test.rb

```diff
@@ -0,0 +1,12 @@
+require 'test-helper'
+
+# UsersControllersTest
+class UsersControllerTest < ActionDispatch::IntegrationTest
+  test 'should get index' do
```

Comment:
- [ ] Following the project requirments, you should create `users_spec.rb` and `posts_spec.rb` in `spec/requests/` directory with `type: request` as shown in the example code:
```ruby
 require 'rails_helper'

 RSpec.describe "Users", type: :request do
   ...
 end
```